README.md

```diff
@@ -1,6 +1,8 @@
 GitLink
 ==========
 
+[![Build status](https://ci.appveyor.com/api/projects/status/g56b4xyl4q0ean8i/branch/develop?svg=true)](https://ci.appveyor.com/project/AArnott/gitlink/branch/develop)
```

Comment:
@GeertvanHorrik yip, GitVersion, GitReleaseManager, and GitReleaseNotes (along with the supporting libraries) all have CI Builds running on AppVeyor.  So think it would make sense to get this up and running there to.  I can create Teams in the GitHub Organisation and assign them to AppVeyor Roles.  That would allow @AArnott to edit the AppVeyor build if required.